packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -332,7 +332,7 @@ class Cursor extends SynchronousCursor implements ITreeSubscriptionCursor {
 		assert(this.innerCursor !== undefined, 0x430 /* Cursor must be current to be used */);
 		return this.innerCursor.exitField();
 	}
-	skipPendingFields(): boolean {
+	override skipPendingFields(): boolean {
```

Comment:
nit: add `public` while you're at it, for consistency?

Reply:
Thats a different flag, well get there :)

Reply:
Haha I'm surprised it's not already in our lint rules.